src/components/shared/WindowTitle.js

```diff
@@ -30,17 +36,18 @@ class WindowTitle extends PureComponent<Props> {
     let title = '';
 
     if (profileName) {
-      title = title.concat(profileName, ' - ');
+      title += profileName + SEPARATOR;
     }
-    title = title.concat(_formatVersion(meta), ' - ');
-    if (meta.oscpu) {
-      title = title.concat(_formatPlatform(meta), ' - ');
+    title += formatProductAndVersion(meta) + SEPARATOR;
+    const os = formatPlatform(meta);
+    if (os) {
+      title += os + SEPARATOR;
```

Comment:
Thanks for the suggestion!
Very soon `formatMetainfoString` will be removed to add icons, but in the window title this will stay the same because there's no icon there. Also I find the code easier to follow here, as we see clearly what will be added to the window title :) So I'll better keep this this way.